src/Sulu/Component/SmartContent/ContentType.php

```diff
@@ -382,13 +382,18 @@ private function getProvider(PropertyInterface $property)
      */
     private function getCurrentPage($pageParameter)
     {
-        if ($this->requestStack->getCurrentRequest() !== null) {
-            $page = $this->requestStack->getCurrentRequest()->get($pageParameter, 1);
-        } else {
-            $page = 1;
+        if ($this->requestStack->getCurrentRequest() === null) {
+            return 1;
+        }
+
+        $page = $this->requestStack->getCurrentRequest()->get($pageParameter, 1);
+        if ($page < 1) {
```

Comment:
Or it could be translated to a 400? Wouldn't that be a better solution? Maybe we should also ask Jens about this. / @chirimoya